api/tests/unit/domain/models/profile_test.js

```diff
@@ -359,6 +359,58 @@ describe('Unit | Domain | Models | Profile', () => {
 
     });
 
+    context('when user has one assessment without existing courseId', () => {
+      it('should return the profile without only assessment of placement in competences', () => {
```

Comment:
Je ne comprends pas trop la phrase dans le it(...) ... ça donne quoi en Français ?